Makefile

```diff
@@ -146,6 +146,7 @@ fauxton: share/www
 
 .PHONY: check
 # target: check - Test everything
+check: COUCH_TEST_ADMIN_PARTY_OVERRIDE = 1
```

Comment:
This should be added to `TEST_OPTS`.

Reply:
I was mistaken, this needs to be exported per relevant target. Pushing a change that should resolve the issue.